evalml/tests/tuner_tests/test_skopt_tuner.py

```diff
@@ -70,6 +70,20 @@ def test_skopt_tuner_basic():
 
     tuner = SKOptTuner(pipeline_hyperparameter_ranges, random_seed=random_seed)
     assert isinstance(tuner, Tuner)
+    first_params = tuner.get_starting_parameters({})
+    assert first_params == {}
+    first_params = tuner.get_starting_parameters(pipeline_hyperparameter_ranges)
+    assert first_params == {
+        "Mock Classifier": {
+            "parameter a": 5,
+            "parameter b": 5.488135039273248,
+            "parameter c": 0,
+            "parameter d": 0,
+            "parameter e": "option a",
+            "parameter f": "option a 💩",
```

Comment:
should I be seeing a poop emoji here???

Reply:
This was work done previously, I didn't change the resutls heh